recipe/patches/disable_explicit_manylinux.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/rust/perspective-python/build.mjs b/rust/perspective-python/build.mjs
+index 78fbd5083..49ca423d5 100644
+--- a/rust/perspective-python/build.mjs
++++ b/rust/perspective-python/build.mjs
+@@ -75,7 +75,7 @@ if (build_wheel) {
+         process.platform === "linux"
+     ) {
+         target =
+-            "--target=x86_64-unknown-linux-gnu --compatibility manylinux_2_28";
++            "--target=x86_64-unknown-linux-gnu --compatibility linux";
```

Comment:
address theseL
-  manylinux_2_28 is too new, pip refuses to install a manylinux_2_28 wheel in the conda build environment
- there are discrepancies between compatible tags printed by `$PYTHON -m pip debug --verbose` in the conda build env, and the manylinux compatibility version detected by maturin